nixos/modules/services/networking/unbound.nix

```diff
@@ -126,11 +188,12 @@ in
         ''}
         touch ${stateDir}/dev/random
         ${pkgs.utillinux}/bin/mount --bind -n /dev/urandom ${stateDir}/dev/random
+        ${optionalString cfg.enableRemoteAccess "${pkgs.unbound}/bin/unbound-control-setup -d ${stateDir}"}
```

Comment:
Shouldn't this be `cfg.package`?

Reply:
Honestly don't know. Saw your new PR, gj :+1:. Keeping this one open til your is merged.